### ISSUE TITLE
Error to present two plots in one panel with word_document2

### ISSUE THREAD

**Issue by sammo3182** [rstudio/bookdown#249]
I tried to present two plots in one panel using the `bookdown::word_document2`. The specification are as following: 

```{r fig.cap='A figure example with the specified aspect ratio, width, and alignment.', fig.show="hold"}
plot(pressure, pch = 19, type = 'b')
plot(pressure$temperature)
```

The codes produce the following error of label once I specified `fig.show="hold"`. The same codes work fine in `pdf_document2` or without the arugment `fig.show="hold"`.
```
Error in parse_fig_labels(x, global) : 
  There are multiple labels on one line: (#fig:unnamed-chunk-1), (#fig:unnamed-chunk-1)
Calls: <Anonymous> ... <Anonymous> -> process_markdown -> parse_fig_labels
Execution halted
```

I'm using the developed version of bookdown (0.2.4).

**Comment by yihui** [comment]
`fig.show='hold'` does not really work for Word output. I just added a warning in the development version of knitr.

**Comment by sammo3182** [comment]
Thank you for the reply. It could be more convenient, though if the compiling of `bookdown::word_document2` can just ignore the `fig.show` option as the `word_document` does even if the function cannot be correctly transferred to MS Word. In that case, users do not need to change the codes in the content but the header to successfully compile the codes and use the cross-reference function of markdown.

**Comment by yihui** [comment]
That is exactly what I did in the development version of knitr.

**Comment by sammo3182** [comment]
Great! Thank you for keeping improving these valuable packages!!

**Comment by yihui** [comment]
The error has been fixed (see #538).
